pennylane/ops/qubit.py

```diff
@@ -533,7 +533,7 @@ class QubitStateVector(Operation):
 
     * Number of wires: None (applied to the entire system)
     * Number of parameters: 1
-    * Gradient recipe: None (uses finite difference)
+    * Gradient recipe: None (integer parameters not supported)
```

Comment:
In this case, the parameter is non-integer, however? It is a complex state vector

Reply:
oh, yes. you are right. How about we make it just 'None'?

Reply:
That sounds good, less ambiguous :slightly_smiling_face: